horovod/runner/launch.py

```diff
@@ -610,7 +611,8 @@ def _run_elastic(args):
                                                 start_timeout=tmout,
                                                 output_filename=args.output_filename,
                                                 run_func_mode=args.run_func is not None,
-                                                nics=args.nics)
+                                                nics=args.nics,
+                                                log_hide_timestamp=args.log_hide_timestamp)
```

Comment:
the command line flag could be named `--log-without-timestamp` and then `log_with_timestamp = ! args.log_without_timestamp`, if it does not have to be a flag it could be `--log-with-timestamp=false`

Reply:
Changed to `--timestamp-output` for consistency with `mpirun`.

Reply:
Also, retain the old arg since it's also used for something else (Horovod internal logging).

Reply:
Do we want to hook into MPI naming convention that much? It is not the best flag name.

Reply:
I agree it's not the best name, but since we need to keep a separate `--log-with-timestamp` flag, I think having a distinct name that MPI users will immediately understand (to help with telling them apart) is beneficial.  What do you think?